Documentation/profiles.md

```diff
@@ -93,20 +93,30 @@ Differences from the default/weak docker profile:
 
 ### Without LSM
 
-TODO
+No LSM is used when SELinux is disabled for a container on a Fedora VM,
+or AppArmor is disabled for a container on a Ubuntu VM.
 
 ### With SELinux
 
-TODO
+The standard SELinux policy of Fedora is used.
+This specially impacts the weak docker profile which has
+`/var/tmp/shared/` as shared folder but cannot access it anymore
+due to different labels.
+The shared folder content is not relabeled with a file label the container is
+allowed to access.
```

Comment:
missing comma? I have difficulties to parse the sentence

Reply:
rephrased it